utils/cli/multisend.go

```diff
@@ -405,26 +406,48 @@ func NewQueryTotalGasCmd() *cobra.Command {
 			}
 
 			now := time.Now().UTC()
+			if clientCtx.Height != 0 {
+				res, err := clientCtx.Client.Block(context.Background(), &clientCtx.Height)
+				if err != nil {
+					fmt.Println("failed to get latest block time")
+					return nil
+				}
+				now = res.Block.Header.Time
+			}
+
 			txtime := now
 			totalgas := int64(0)
+			numRelays := int64(0)
 			sequence, _ := getSequence(account)
 			layout := time.RFC3339
-			for now.Sub(txtime) < 24*time.Hour {
+			for now.Sub(txtime) < 12*time.Hour {
```

Comment:
update cli help to 12h

Reply:
done. see https://github.com/lavanet/lava/commit/5b44b0fc38bc2ab755612c85c83be4c287d25e9d